lib/chatty/connection.ex

```diff
@@ -119,10 +144,24 @@ defmodule Chatty.Connection do
     {:noreply, %{state | sock: nil, last_message_time: nil}}
   end
 
+  def handle_info({:ssl_error, sock, reason}, %{sock: sock} = state) do
```

Comment:
Well, the `handle_info` functions are based upon what the :ssl application sends, I just saw the stacktraces when messages weren't handled and wrote those functions according to what the module was expecting…

Writing a helper just to write `irc_close(sock)` instead of `:ssl.close(sock)` and `:gen_tcp(sock)`
seems a bit overkill in my opinion…